### PR TITLE
[#29] regression fix for launching oracle database and pipeline stage fix

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

File: irods_docker_files/docker_cmds_utilities.py [hunk 1]
```diff
@@ -133,7 +137,8 @@ def run_database(database_type, database_container, alias_name, network_name):
     database_alias = 'database.example.org'
     if database_type == 'oracle':
         database_alias = 'oracle.example.org'
-        run_cmd = ['docker', 'run', '-d', '--rm',  '--name', database_container, '--shm-size=1g', '-e', 'ORACLE_PWD=testpassword', 'oracle/database:11.2.0.2-xe']
+        run_cmd = ['docker', 'run', '-d', '--rm',  '--name', database_container, '-h', database_alias, '--shm-size=1g', '-e', 'ORACLE_PWD=testpassword', 'oracle/database:11.2.0.2-xe']
```
Comment (trel) [inline irods_docker_files/docker_cmds_utilities.py:140]: can use 'database_image' variable from above?

Comment (jassigill2000) [reply irods_docker_files/docker_cmds_utilities.py:139]: i have added the name of the oracle image in the configuration.py file just like other database image names.

Comment (trel) [reply irods_docker_files/docker_cmds_utilities.py:139]: i see it up top - please use it in line 139

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml [hunk 2]
```diff
@@ -206,7 +206,7 @@
         }
     }
     if(PARAMETER_RUN_TOPOLOGY_RESOURCE_TESTS == &quot;true&quot;) {
-        stage(&quot;Install and Run Topology Tests&quot;) {
+        stage(&quot;Install and Run Topology Tests on Resource&quot;) {
```
Comment (trel) [inline jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml:209]: Consumer

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml [hunk 3]
```diff
@@ -218,7 +218,7 @@
         }
     }
     if(PARAMETER_RUN_TOPOLOGY_RESOURCE_TESTS == &quot;true&quot; &amp;&amp; PARAMETER_ENABLE_SSL == &quot;true&quot;) {
-        stage(&quot;Install and Run Topology Tests With SSL&quot;) {
+        stage(&quot;Install and Run Topology Tests With SSL on Resource&quot;) {
```
Comment (trel) [inline jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml:221]: Consumer

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml (hunk 2 above)
Comment (trel) [reply jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml:209]: Consumer

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml (hunk 3 above)
Comment (trel) [reply jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml:221]: Consumer

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml (hunk 2 above)
Comment (jassigill2000) [reply jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml]: Done

File: jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml (hunk 3 above)
Comment (jassigill2000) [reply jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml]: Done

**Pull request merged**